src/components/collection/Summary.js

```diff
@@ -91,6 +115,43 @@ function Summary({
     setOpen(false);
   };
 
+  const handleFavorite = () => {
+    updateFavoriteItem({
```

Comment:
It's actually editing the member right? Perhaps we can create a whole new mutations in queryclient, which would handle the whole process. For instance `addItemInFavorite` and `removeItemInFavorite` given the member's id and the item id, and these mutations would create the arrays correctly. I think it will nicely simplify the frontend.

Reply:
I'm not very sure about this one: As we use the endpoint 'edit member' anyways, with new mutation, we still need to pass in the old favoriteItems, and do the operations to the array. It's a bit of trouble to set the 'extra' field in mutation functions, so I would prefer keep it in frontend. Let me know what you think!